packages/core/src/amazonqFeatureDev/errors.ts

```diff
@@ -26,12 +34,20 @@ export class TabIdNotFoundError extends ToolkitError {
     }
 }
 
+/**
+ * Error thrown when the UI panel fails to load.
+ * @extends ToolkitError
+ */
 export class PanelLoadError extends ToolkitError {
     constructor() {
         super(`${featureName} UI panel failed to load`, { code: 'PanelLoadFailed' })
     }
 }
 
+/**
+ * Error thrown when the workspace folder is not found.
+ * @extends ToolkitError
```

Comment:
why are we adding `@extends` jsdoc directives? this is a typescript project. there is already a `extends` declaration in the code below.